services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/common/AttachmentValidator.kt

```diff
@@ -5,48 +5,25 @@ import java.util.regex.Pattern
 import mu.KotlinLogging
 import org.apache.commons.io.FilenameUtils.getExtension
 import org.apache.commons.io.FilenameUtils.removeExtension
-import org.apache.tika.Tika
 import org.springframework.web.multipart.MultipartFile
 
 private val logger = KotlinLogging.logger {}
 
 private val supportedFiletypes =
-    mapOf(
-        "application/pdf" to setOf("pdf"),
-        "application/msword" to setOf("docx", "doc"),
-        "image/jpeg" to setOf("jpg", "jpeg"),
-        "image/png" to setOf("png"),
-        "image/vnd.dwg" to setOf("dwg", "dws")
+    setOf(
+        "pdf",
+        "jpeg",
```

Comment:
We probably want to allow jpg as well, both are used extensively in the wild.

Reply:
Added.